paper-server/patches/sources/net/minecraft/world/level/block/entity/BrushableBlockEntity.java.patch

```diff
@@ -1,5 +1,66 @@
 --- a/net/minecraft/world/level/block/entity/BrushableBlockEntity.java
 +++ b/net/minecraft/world/level/block/entity/BrushableBlockEntity.java
+@@ -49,6 +_,7 @@
+     @Nullable
+     public ResourceKey<LootTable> lootTable;
+     public long lootTableSeed;
++    private int oldBrushCount; // PaperMC - store old brush count for EntityChangeBlockEvent reversion
```

Comment:
ended up going with a field here, other option was an extra parameter in the brushingComplete, method, and I'm not sure if I like that diff as much, felt it was easier to insert as a field here.